news/static/news/css/admin-event-tickets.css

```diff
@@ -0,0 +1,4 @@
+.input-monospace {
+    font-family: 'Courier New', Courier, monospace;
+    width: 100%;
+}
```

Comment:
Please add a new line at the end of the file

Reply:
Fixed! Why should I add it, though?

Reply:
It's just a convention; see the answers to [this question](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file) 🙂